cms/server/admin/templates/activate_dataset.html

```diff
@@ -94,7 +94,7 @@ <h1>Preview dataset change for {{ task.title }} (<a href="{{ url("task", task.id
         Public score: {{ c.old_public_score }} -&gt; {{ c.new_public_score }}<br/>
         {% endif %}
         {% if c.old_ranking_score_details is not none or c.new_ranking_score_details is not none %}
-        Ranking score: {{ json.dumps(c.old_ranking_score_details) }} -&gt; {{ json.dumps(c.new_ranking_score_details) }}<br/>
+        Ranking score: {{ c.old_ranking_score_details|tojson }} -&gt; {{ c.new_ranking_score_details|tojson }}<br/>
```

Comment:
Is `tojson` builtin? Asking since we have `to_language`, `make_timestamp`, `tojson`...

Reply:
`tojson` is a builtin, we can't change. I am not opposed to renaming the other ones to `tolanguage` and `totimestamp` if you want me to.

Reply:
Nah, not now.